Advanced_Projects/MultiAgent_Debate_DAG/nodes/judge_node.py

```
@@ -0,0 +1,43 @@
+        model="mistralai/Mistral-7B-Instruct-v0.2",
+        max_tokens=200,
+        temperature=0.7,
+        top_k=50,
+        top_p=0.9,
+    )
+
+    # Parse the output string cleanly
+    output = response['choices'][0]['text'].strip()
+
+    # Parse lines to avoid duplication
+    summary_line = next((l for l in output.split('\n') if l.startswith("[Judge] Summary:")), "").replace("[Judge] Summary:", "").strip()
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)